Tasks/ExtractFilesV1/extractfilestask.ts

```diff
@@ -8,6 +8,7 @@ import os = require('os');
 var archiveFilePatterns: string[] = tl.getDelimitedInput('archiveFilePatterns', '\n', true);
 var destinationFolder: string = path.normalize(tl.getPathInput('destinationFolder', true, false).trim());
 var cleanDestinationFolder: boolean = tl.getBoolInput('cleanDestinationFolder', false);
+var overwriteExistingFiles: boolean = tl.getBoolInput('overwriteExistingFiles', false);
```

Comment:
Can we use `let` or `const` here?

Reply:
Yeah, we can use `const` here, `var` were used just for consistency with other variables declaration for input fields. Should I change it just for new variable only or for all of them?

Reply:
Thanks @anton-erofeev for the point - let's just leave it as is for consistency here